Code/Framework/AzCore/AzCore/Jobs/Job.h

```diff
@@ -61,7 +72,7 @@ namespace AZ
          *          The valid range is -128 (lowest priority) to 127 (highest priority), the default is 0,
          *          and jobs with equal priority values will be run in the same order as added to the queue.
          */
-        Job(bool isAutoDelete, JobContext* context, bool isCompletion = false, AZ::s8 priority = 0);
+        Job(bool isAutoDelete, JobContext* context, bool isCompletion = false, Priority priority = Priority::MEDIUM);
 
```

Comment:
Do we care about backward compat?  Maybe we could make a constructor that maps the priority into one range or even just sets it all to medium? 

Or don't think anyones using this?

Reply:
Well this `Job` constructor is not normally invoked by users.
Jobs are normally created via the `AZ::CreateJobFunction` wrapper

Reply:
Nobody is using this in my sweep and the CreateJobFunction doesn't even expose it

Reply:
sounds fine to me then